applications/crossbar/src/crossbar_app.erl

```diff
@@ -23,6 +23,8 @@
 %%--------------------------------------------------------------------
 -spec start(application:start_type(), any()) -> startapp_ret().
 start(_StartType, _StartArgs) ->
+    _ = kazoo_bindings:bind(kapps_maintenance:binding(migrate), crossbar_maintenance, migrate),
```

Comment:
@jamhed we tickie all atoms, so `'crossbar_maintenance'` and `'migrate'` would be appropriate.